### DISCUSSION TITLE
评论者的归属地可以简繁转换或做其他修改么？

### DISCUSSION THREAD

**Discussion by fez7yoe** [walinejs/waline#1122] [Q&A] [answered]
我知道评论者的 UA 跟归属地可以隐藏，但归属地的字可修改么？
例如简转繁

**Comment by lizheming** [comment, marked as answer]
暂时不支持。
归属地的文案修改应该比较复杂，目前用的是这个 IP 归属地查询的库 https://github.com/yumeiqiang/dy-node-ip2region/ 需要修改归属地的字的话需要自定义一份 IP 归属地的数据库了。
